iroha_data_model/src/query.rs

```diff
@@ -539,11 +545,42 @@ pub mod peer {
     }
 }
 
+pub mod transaction {
+    //! Queries related to `Transaction`.
+
+    use iroha_derive::Io;
+    use parity_scale_codec::{Decode, Encode};
+    use serde::{Deserialize, Serialize};
+
+    use crate::account::prelude::AccountId;
+    use crate::expression::EvaluatesTo;
+
+    /// `FindTransactionsByAccountId` Iroha Query will find all transaction included in blockchain
+    /// for the account
+    #[derive(Clone, Debug, Io, Serialize, Deserialize, Encode, Decode, PartialEq, Eq)]
+    pub struct FindTransactionsByAccountId {
+        /// `AccountId` under which transactions should be found.
+        pub account_id: EvaluatesTo<AccountId>,
+    }
+
+    impl FindTransactionsByAccountId {
+        ///Default `FindTransactionsByAccountId` constructor.
```

Comment:
```suggestion
        ///Default [`FindTransactionsByAccountId`] constructor.
```

Reply:
Only this way it will be properly linked in docs.

Reply:
See https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html

Reply:
Fixed